TYPO3.Media/Classes/TYPO3/Media/ViewHelpers/ImageViewHelper.php

```diff
@@ -104,14 +104,14 @@ public function initializeArguments()
         $this->registerTagAttribute('alt', 'string', 'Specifies an alternate text for an image', true);
         $this->registerTagAttribute('ismap', 'string', 'Specifies an image as a server-side image-map. Rarely used. Look at usemap instead', false);
         $this->registerTagAttribute('usemap', 'string', 'Specifies an image as a client-side image-map', false);
-        // @deprecated since 2.0 use the "image" argument instead
-        $this->registerArgument('asset', 'TYPO3\Media\Domain\Model\AssetInterface', 'The image to be rendered - DEPRECATED, use "image" argument instead', false);
+        // @deprecated since 2.1 use "asset" argument instead
+        $this->registerArgument('image', 'TYPO3\Media\Domain\Model\ImageInterface', 'The image to be rendered - DEPRECATED, use "asset" argument instead', false);
```

Comment:
Wait, why are we changing this again? IMHO that is first of all pretty bad for users and second why should we allow assets here?
IMHO that waters down the purpose of the ImageViewHelper even more, I would rather create a thumbnail VH for this purpose.